dbms/src/Storages/DeltaMerge/DeltaMerge.h

```diff
@@ -469,8 +469,19 @@ class DeltaMergeBlockInputStream final : public SkippableBlockInputStream
                     output_columns[column_id]->reserve(max_block_size);
             }
             for (size_t column_id = 0; column_id < num_columns; ++column_id)
-                output_columns[column_id]->insertRangeFrom(*cur_stable_block_columns[column_id], final_offset, final_limit);
-
+                if (cur_stable_block_columns[column_id]->isColumnConst())
+                {
+                    for (size_t index = 0; index < final_limit; ++index)
+                    {
+                        Field value;
+                        cur_stable_block_columns[column_id]->get(0, value);
```

Comment:
I think we should get value from `final_offset + index`?

Reply:
it's a const column, just has one element in column. Thus I think we can always just get the index = 0 value.